app/src/main/java/com/hedvig/app/feature/chat/ui/ChatInputView.kt

```diff
@@ -229,10 +252,12 @@ class ChatInputView : FrameLayout {
                 SingleSelectChoiceType.SELECTION -> sendSingleSelect(value)
                 SingleSelectChoiceType.LINK -> singleSelectLink(value)
             }
-
         }
 
         singleSelectContainer.addView(singleSelectButton)
+
+        singleSelectContainer.measure(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT)
+        chatRecyclerView.updatePadding(bottom = chatRecyclerViewInitialPadding + singleSelectContainer.measuredHeight - measureTextInput())
```

Comment:
Does this overwrite padding we've added through system window insets?

Reply:
Yes. But ut works as intended now as far as i can tell, after a minor fix

Reply:
There might be some potential for simplification here, but lets keep it as is for now.